src/components/Identity/util.js

```diff
@@ -43,4 +43,11 @@ const normalizeCustomerIds = customerIds => {
   }, {});
 };
 
-export { validateCustomerIds, normalizeCustomerIds };
+const updateCustomerIdState = (currentState, newState) => {
+  return {
+    ...currentState,
+    ...newState
+  };
+};
```

Comment:
I don't see this being used anywhere.

Reply:
I probably would advise removing it anwyay as it's simple enough and generic enough that I don't think it warrants being a utility.